wiki/osu!dev_Discord_server/pt-br.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - development
+---
+
+# Servidor do Discord osu!dev 
```

Comment:
```suggestion
# Servidor de Discord do osu!dev 
```
Suggestion for ease of reading. `Servidor do Discord do osu!dev` works as well but repeating `do` doesn't sound as nice.